esridump/dumper.py

```diff
@@ -431,7 +432,7 @@ def __iter__(self):
                     # pause every number of "requests_to_pause", that increase the probability for server response
                     if query_index % self._requests_to_pause == 0:
                         time.sleep(self._pause_seconds)
-                        self._logger.info("pause for {0} seconds", self._pause_seconds)
+                        self._logger.info("pause for {0} seconds".format(self._pause_seconds))
```

Comment:
```suggestion
                        self._logger.info("pause for %s seconds", self._pause_seconds)
```

Python loggers can do sprintf style formatting.